packages/thirdweb/src/utils/encoding/hex.ts

```diff
@@ -503,3 +504,19 @@ export function toHex(
       return uint8ArrayToHex(value, opts);
   }
 }
+
+/**
+ * Generates a random hexidecimal string.
+ * @param length Length of the Hex to generate.
+ * @returns The generated hexadecimal string.
+ * @example
+ * @example
+ * ```ts
+ * const randomHex = generateRandomHex(16);
+ * console.log(randomHex); // Outputs a 16-character hexadecimal string, e.g., "a3f7b1c482e8f440"
+ * ```
+ * @util
+ */
+export function generateRandomHex(length: number): string {
```

Comment:
i think we already have a `randomBytes32()` in `utils/uuid.ts` - does it do what you need if we just parametrize the legth? would rather have a single function for this